benchmarks/htap/queries/05.sql.template

```diff
@@ -26,6 +27,7 @@ where
     and r_name = 'Europe'
     and o_entry_d >= '$begin_date'
     and o_entry_d < '$end_date'
+    and ol_delivery_d >= '$min_date'
```

Comment:
Are these changes compatible with TPCH specs?

Reply:
no but neither are the queries ;) tpc-h doesn't support > 7 years of data so i'm making that we only do 7 years

Reply:
Wunderbar